src/core_plugins/console/public/src/sense_editor/mode/worker.js

```diff
@@ -0,0 +1,2 @@
\":\"),Object.hasOwnProperty.call(n,t)&&c(\'Duplicate key \"\'+t+\'\"\'),n[t]=i(),f(),\"}\"===e)return a(\"}\"),n;a(\",\"),f()}}c(\"Bad object\")};i=function(){switch(f(),e){case\"{\":return m();case\"[\":return function(){const t=[];if(\"[\"===e){if(a(\"[\"),f(),\"]\"===e)return a(\"]\"),t;for(;e;){if(t.push(i()),f(),\"]\"===e)return a(\"]\"),t;a(\",\"),f()}}c(\"Bad array\")}();case\'\"\':return h();case\"-\":return u();default:return e>=\"0\"&&e<=\"9\"?u():function(){switch(e){case\"t\":return a(\"t\"),a(\"r\"),a(\"u\"),a(\"e\"),!0;case\"f\":return a(\"f\"),a(\"a\"),a(\"l\"),a(\"s\"),a(\"e\"),!1;case\"n\":return a(\"n\"),a(\"u\"),a(\"l\"),a(\"l\"),null}c(\"Unexpected \'\"+e+\"\'\")}()}};let g=function(){for(f(),function(){switch(e){case\"G\":return a(\"G\"),a(\"E\"),a(\"T\"),\"GET\";case\"H\":return a(\"H\"),a(\"E\"),a(\"A\"),a(\"D\"),\"HEAD\";case\"D\":return a(\"D\"),a(\"E\"),a(\"L\"),a(\"E\"),a(\"T\"),a(\"E\"),\"DELETE\";case\"P\":switch(a(\"P\"),e){case\"U\":return a(\"U\"),a(\"T\"),\"PUT\";case\"O\":return a(\"O\"),a(\"S\"),a(\"T\"),\"POST\";default:c(\"Unexpected \'\"+e+\"\'\")}break;default:c(\"Expected one of GET\/POST\/PUT\/DELETE\/HEAD\")}}(),p(),function(){let t=\"\";for(;e&&\"\\n\"!=e;)t+=e,a();\"\"==t&&c(\"Missing url\")}(),p(),d(),p(),\"{\"==e&&m(),p(),d(),p();\"{\"==e;)m(),p(),d(),p()},w=function(){for(;\"#\"==e;){for(;e&&\"\\n\"!==e;)a();f()}};return function(i,o){let c;return r=i,t=0,n=[],a(),function(){for(;e&&\"\"!=e;)if(f(),e)try{if(w(),f(),!e)continue;g(),f()}catch(e){s(\"error\",e.message);const n=r.substr(t).search(\/^POST|HEAD|GET|PUT|DELETE\/m);if(n<1)return;l(t+n)}}(),f(),e&&s(\"error\",\"Syntax error\"),c={annotations:n},\"function\"==typeof o?function t(e,n){let r,i,s=e[n];if(s&&\"object\"==typeof s)for(r in s)Object.hasOwnProperty.call(s,r)&&(void 0!==(i=t(s,r))?s[r]=i:delete s[r]);return o.call(e,n,s)}({\"\":c},\"\"):c}}),ace.define(\"sense_editor\/mode\/worker\",[\"require\",\"exports\",\"module\",\"ace\/lib\/oop\",\"ace\/worker\/mirror\",\"sense_editor\/mode\/worker_parser\"],function(t,e){const n=t(\"ace\/lib\/oop\"),r=t(\"ace\/worker\/mirror\").Mirror,i=t(\"sense_editor\/mode\/worker_parser\"),o=e.SenseWorker=function(t){r.call(this,t),this.setTimeout(200)};n.inherits(o,r),function(){this.id=\"senseWorker\",this.onUpdate=function(){const t=this.doc.getValue();let e,n;try{n=i(t)}catch(t){return e=this.charToDocumentPosition(t.at-1),void this.sender.emit(\"error\",{row:e.row,column:e.column,text:t.message,type:\"error\"})}for(let t=0;t<n.annotations.length;t++)e=this.charToDocumentPosition(n.annotations[t].at-1),n.annotations[t].row=e.row,n.annotations[t].column=e.column;this.sender.emit(\"ok\",n.annotations)},this.charToDocumentPosition=function(t){let e=0;const n=this.doc.getLength(),r=this.doc.getNewLineCharacter().length;if(!n)return{row:0,column:0};let i,o=0;for(;e<n;){const n=(i=this.doc.getLine(e)).length+r;if(o+n>t)return{row:e,column:t-o};o+=n,e+=1}return{row:e-1,column:i.length}}}.call(o.prototype)});';
```

Comment:
Did you mean to revert this?

Reply:
yeah.  this was not working as I thought it did when I tested it.  Working on better solution.  Weirdly raw-loader adds use strict to the top of the file and this seems to break things AFAICT.

Reply:
That's probably because it was being run through babel, if you use `require('!!raw-loader!./worker.js')` the default loaders for `.js` files will not be applied